ios/Rainbow.xcodeproj/project.pbxproj

```diff
@@ -9,6 +9,7 @@
 /* Begin PBXBuildFile section */
 		0299CE7B2886202800B5C7E7 /* NotificationService.m in Sources */ = {isa = PBXBuildFile; fileRef = 0299CE7A2886202800B5C7E7 /* NotificationService.m */; };
 		0299CE7F2886202800B5C7E7 /* ImageNotification.appex in Embed App Extensions */ = {isa = PBXBuildFile; fileRef = 0299CE772886202800B5C7E7 /* ImageNotification.appex */; settings = {ATTRIBUTES = (RemoveHeadersOnCopy, ); }; };
+		0A75490AB3C4D644E34D9BE0 /* libPods-PriceWidgetExtension.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 2D9D6426C5AE0E210FF6C21A /* libPods-PriceWidgetExtension.a */; };
```

Comment:
I don't ~think we need any changes to this file

Reply:
i didnt do anything it was autogenerated afaik